File: C++/0035. Reversal.cpp
// we have defined the necessary header files here for this problem.
// If additional header files are needed in your program, please import here.
#include <algorithm>
int main()
{  
  // please define the C++ input here. For example: int a,b; cin>>a>>b;;  
  // please finish the function body here.  
  // please define the C++ output here. For example:cout<<____<<endl; 
    string s;
    string res="";
    while(cin>>s){
        reverse(s.begin(),s.end());
        if(res!=""){
            res += " "+s;
        }
        else{
            res = s;
        }
    }
    cout<<res<<endl;
    return 0;
}

File: Python/0035. Reversal.py
# If you need to import additional packages or classes, please import here.

def func():
    # please define the python3 input here. 
    # For example: a,b = map(int, input().strip().split())
    # please finish the function body here.
    # please define the python3 output here. For example: print().
    while True:
        try:
            print((' ').join([s[::-1] for s in list(map(str,input().split()))]))
            
        except EOFError:
            break
    
if __name__ == "__main__":
    func()
